### PR TITLE
dhd: Immediately fail Android builds on OBS if a command fails

### PR DESCRIPTION
**Pull request by Thaodan**
Run the Android build commands with set -e/errexit so that in case on
them fails the shell directly exits instead of silently continuing.

### CODE REVIEW

**Comment by sledges** [comment]
Old `ubu-chroot` (before https://github.com/sailfishos/android-tools-hadk/pull/7) will work incorrectly on OBS when invoked with `sh -c`.
Could you provide a safe-guard check of which `ubu-chroot` is being used?

**Comment by sledges** [comment]
Nitpick: typo: to -> too.

Please fall back to the old cmdline so the transition is less painful for everybody:)

**Comment by Thaodan** [comment]
> Nitpick: typo: to -> too.
> 
> Please fall back to the old cmdline so the transition is less painful for everybody:)

Adjusted please review.

File: droid-hal-device.inc [hunk 1]
```diff
@@ -399,8 +399,12 @@ else
 fi
 
 # Hadk style build of android on OBS 
+if ubu-chroot -v ; then
+   sh="sh -c"
```
Comment (sledges) [inline droid-hal-device.inc:403]: LGTM, worth commenting that argument parsing has been introduced together with version reporting.

**Comment by sledges** [comment]
nit: obs->OBS (commit, body, and PR desc)

**Comment by sledges** [comment]
in case on them fails->
in case one of them fails

**Comment by sledges** [comment]
> in case on them fails-> in case one of them fails

one them -> one of them

**Review by sledges — CHANGES_REQUESTED** [review]
Thanks for your patience:)

**Pull request merged**